plugins/module_utils/network/ios/ios.py

```diff
@@ -190,6 +190,8 @@ def _get_number(name):
         if_type = "TwentyFiveGigE"
     elif name.lower().startswith("hu"):
         if_type = "HundredGigE"
+    elif name.lower().startswith("se"):
```

Comment:
The normalization method seems to exist in multiple places. We should have just one. But I understand that this might be out of scope for this PR since other modules would need their imports to be updated.

Reply:
@NilashishC yes, There are multiple places where similar operations are done, I would take it as a note a push the change in a different PR. Thanks